### ISSUE TITLE
Enable moderation workflow for recent submissions

### ISSUE THREAD

**Issue by igrigorik** [igrigorik/vimgolf#183]
From @urticadioica in https://github.com/igrigorik/vimgolf/issues/170#issuecomment-269874140: 

> As for preventing cheats, that would be nice, and I'd agree with the proposal no hesitation if there weren't an interesting tactic it would wipe out. But ultimately, there's no way to prevent cheats short of server-side verification. What would really help help catch cheaters quickly is a recent solution page, either showing all recent solutions (though the volume might be too high), or just new first place entries. If there's a new first place, either we've got a cheater, or someone has found something really cool. Either way I want to know. (It's a bit disappointing when I don't see a new discovery until months later.) Now that everyone's helping work on the server, this would be my top wishlist item.

Seems like a great feature to enable for admins/moderators.

**Comment by Smylers** [comment]
> > What would really help ... is a recent solution page ... showing new first place entries. If there's a new first place, either we've got a cheater, or someone has found something really cool.
>
> Seems like a great feature to enable for admins/moderators.

Why only administrators? I'd always be interested in looking at a new lower-scoring solution on an old challenge.

Take [Remove Accent off the Letter](http://www.vimgolf.com/challenges/4d1c7ee635b40650b8000203) for example: 2½ years went past between @urticadioica's solution and mine, and then it turns out another year for somebody else to improve on that — except I didn't know about that till just now.
For the first few months after submitting, I intermittently checked that page for new solutions or comments, but there weren't any; it's only looking up the URL now for this comment that I learn of the improvement posted last October.

So it'd be great to have a list somewhere of newest first-place entries in reverse-chronological order, limited to showing one per challenge (so that recent challenges don't dominate the list with entries that have already been bettered).

It could even be the main list of challenges on the homepage that does this:

* Put any challenges with no entries first.
* Then for challenges with entries, list them by reverse-chron of first-place entry, perhaps also showing what the current lowest score is and its date submitted.

Recent challenges would still be towards the top of the homepage, but an old challenge with a new first-place entry would jump up the list.

**Comment by igrigorik** [comment]
Yep, an open page with list of most recent entries seems reasonable. Thanks
for the feedback!
